src/lib/packages.ts

```diff
@@ -68,10 +69,10 @@ export namespace apply {
    * Logger definitions
    */
   export interface Options {
-    before: (packages: string[]) => string;
-    beforeEach: (packageName: string) => string;
-    afterEach: (packageName: string, error?: Error) => string;
-    after: (packages: string[], errors: Error[]) => string;
+    before(packages: string[]): string;
```

Comment:
Although the two are equivalent, the method form usually implies that function works in conjunction with a `this` context, whereas the function property form implies that the function works anywhere.

Reply:
noted. I'm using the oclif lint rules without modification for the moment. it insisted on this form.